shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -34,6 +34,13 @@
 // returns kInvalidFirstRect, iOS will not show the IME candidates view.
 const CGRect kInvalidFirstRect = {{-1, -1}, {9999, 9999}};
 
+// Position FlutterTextInputView outside of the screen (if scribble is disabled).
```

Comment:
What effect does this have on IME display? Or do we do all of that via Flutter UI?

Reply:
I can confirm that IME still works. Below is IME with software and hardware keyboards: 

<img src="https://github.com/flutter/engine/assets/41930132/87bbef85-33a1-4689-85da-535d616e7d91" width="200">

<img src="https://github.com/flutter/engine/assets/41930132/4a95c7db-afae-4abe-9718-a8f88e8d8877" width="200">

Reply:
Ah I tested it again, and found iOS 16 and 17 have different behaviors for IME. Without the fix, iPhone shows IME on top ([issue here](https://github.com/flutter/flutter/issues/132551)). The current behavior is bad, but this PR will hide IME, which is even worse. 

Even after putting the UIInputView to overlap with text doesn't 100% resolve the problem - the IME is still slightly off. Will dig into this more.